lib/rules/require-meta-schema.js

```diff
@@ -33,27 +44,33 @@ module.exports = {
     }
 
     let contextIdentifiers;
-    let hasEmptySchema = false;
+    const metaNode = info.meta;
     let schemaNode;
 
+    // Options
+    const requireSchemaPropertyWhenOptionless = !context.options[0] || context.options[0].requireSchemaPropertyWhenOptionless;
+
+    let hasEmptySchema = false;
+
     return {
       Program (ast) {
         contextIdentifiers = utils.getContextIdentifiers(context, ast);
 
-        const metaNode = info.meta;
         schemaNode =
           metaNode &&
           metaNode.properties &&
           metaNode.properties.find(p => p.type === 'Property' && utils.getKeyName(p) === 'schema');
 
         if (!schemaNode) {
-          context.report({
-            node: metaNode,
-            messageId: 'missing',
-            fix (fixer) {
-              return utils.insertProperty(fixer, metaNode, 'schema: []', sourceCode);
-            },
-          });
+          if (requireSchemaPropertyWhenOptionless) {
+            context.report({
+              node: metaNode,
+              messageId: 'missing',
+              fix (fixer) {
+                return utils.insertProperty(fixer, metaNode, 'schema: []', sourceCode);
```

Comment:
is it likely to change the runtime behavior?
before fixing: skip validating
after fixing: no options allowed.

or it can be a suggestion?